src/main/java/fr/uge/dummychargepoint/socket/ConfigurationServer.java

```diff
@@ -20,35 +23,53 @@ public ConfigurationServer(InetSocketAddress address) {
   public void onOpen(WebSocket webSocket, ClientHandshake clientHandshake) {
     Objects.requireNonNull(webSocket);
     Objects.requireNonNull(clientHandshake);
-    var remote = webSocket.getRemoteSocketAddress();
-    LOGGER.info("New connection to: {}", remote);
+    synchronized (lock) {
+      if (client != null) {
+        webSocket.closeConnection(1, "Server is busy.");
```

Comment:
what is `1` ? give a name to a variable to explain, avoid magic numbers

Reply:
Yep yep yep, you are totally right, silly me

I will find the correct error code for "server busy"

Reply:
Done, [bd13aae](https://github.com/LastProject-ESIEE/dummy-chargepoint/pull/6/commits/bd13aae7756d248f1c977b788534d6425c67067c)